### PR TITLE
[RFC][NI] Migrate mobile menu from self-service to uniq

### PR DESCRIPTION
**Pull request by montoias**
NI

### CODE REVIEW

File: addon/styles/partials/_uni-datepicker.scss [hunk 1]
```diff
@@ -90,16 +91,17 @@ $box-size: 42px;
     &:last-child {
       @include justify-content-start();
 
-      [role="button"] {
+      button {
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker.scss:94]: Haven't we migrated the buttons to divs?

File: tests/integration/components/uni-header-mobile-menu-button-test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('uni-header-mobile-menu-button', 'Integration | Component | uni header mobile menu button', {
+  integration: true
+});
+
+test('it renders', function(assert) {
```
Comment (AndreJoaquim) [inline tests/integration/components/uni-header-mobile-menu-button-test.js:8]: No need to make tests but at least make them pass xD

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-datepicker.scss (hunk 1 above)
Comment (montoias) [reply addon/styles/partials/_uni-datepicker.scss:94]: this is from the plugin. @mmelo switched them for that reason, but the plugin is rendering buttons and not divs with presentation button

File: tests/integration/components/uni-header-mobile-menu-button-test.js (hunk 2 above)
Comment (montoias) [reply tests/integration/components/uni-header-mobile-menu-button-test.js:8]: XD

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**